utils/CleanDatabases.ps1

```diff
@@ -0,0 +1,26 @@
+$DatabaseUrl = "http://localhost:8083"
+$DatabasePath = "C:\RavenDBv3\Data"
+$WaitMinutes = 0
+$ignore = @( "system", "databases" )
+
+function TC-Write([string]$name) {
+	Write-Output "##teamcity[message text='$name']"
+}
+
+TC-Write "Running RavenDB Database Cleaner"
+TC-Write "RavenDB URL: $DatabaseUrl"
+TC-Write "RavenDB Data Path: $DatabasePath"
+
+$databases = Get-ChildItem $DatabasePath -Directory `
+| Where-Object { $ignore -notcontains $_.Name } `
+| Where-Object { $_.CreationTimeUtc -lt (([DateTime]::UtcNow).AddMinutes(-$WaitMinutes)) }
+
+
+foreach($database in $databases) {
+    TC-Write "Deleting $database"
+    $deleteUrl = "$($DatabaseUrl)/admin/databases/$([Uri]::EscapeDataString($database.Name))?hard-delete=true"
```

Comment:
this is problematic, IMO. This what should happens at the end of each test when the the DocStore is disposed however it can fail for an an unknown reason.

What is telling us that it should work here?